blog/2024-01-07-kvrocks-2023-in-review/index.md

```diff
@@ -0,0 +1,67 @@
+---
+slug: kvrocks-2023-in-review
+title: Apache Kvrocks 2023 In Review
+---
+
+The year 2023 is a small milestone for the community in open source, and it is also a new starting point. **Kvrocks successfully graduated from the incubator to become an Apache top-level project in June, which is a great affirmation for the community's health and sustainability**. At the same time, there are also some exciting progress in the project and community.
+
+Before we start, we would like to thanks all the contributors and users who have been supporting Apache Kvrocks. We can't make it without you!
+
+<!--truncate-->
+
+### Project Retrospective
+
+We collected a lot of feedback from users and developers in the discussion: [2023 Planning(Want your feedback)](https://github.com/apache/kvrocks/discussions/1226), some of them have been resolved in 2023, and some of them are still in the plan list.
+
+**The key progresses in 2023:**
+
+- JSON data structure
+- BloomFilter data structure
+- Watch/Unwatch command
+- Lua Functions
+- Replication with TLS
+- 65 bits expire time and size
```

Comment:
```suggestion
- Allow to enable 64 bits expiration time(precision is millisecond instead of second), and the length of List/Hash/Set/ZSET/Stream can be also extended from 32 to 64 bits.
```
@mapleFU Is this more clear than before?

Reply:
Yeah I think so